src/pytypes/onefuzztypes/_monkeypatch.py

```diff
@@ -0,0 +1,50 @@
+# TODO: Remove once `smart_union` like support is added to Pydantic
+#
+# Written by @PrettyWood
+# Code from https://github.com/samuelcolvin/pydantic/pull/2092
+#
+# Original project licensed under the MIT License.
+
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Union
+
+from pydantic.fields import ModelField
+from pydantic.typing import get_origin
+
+if TYPE_CHECKING:
```

Comment:
Can we document why we need this, e.g. "define aliases for readability".

Reply:
I simplified it to not provide our own types, just use the ones pydantic provides.